cmd/podman/images/build.go

```diff
@@ -290,6 +292,13 @@ func build(cmd *cobra.Command, args []string) error {
 
 	report, err := registry.ImageEngine().Build(registry.GetContext(), containerFiles, *apiBuildOpts)
 	if err != nil {
+		exitCode := buildahCLI.ExecErrorCodeGeneric
+		if ee, ok := (errors.Cause(err)).(*exec.ExitError); ok {
+			if w, ok := ee.Sys().(syscall.WaitStatus); ok {
+				exitCode = w.ExitStatus()
+			}
```

Comment:
There is no need for this, you can directly call `ee.ExitCode()`

Reply:
Nice.

Reply:
@Luap99 Fixed in latest commit however I think go versions older than `1.12` does not contains `ExitCode()` but I think we are not going to be using `1.12` so we should be good.